app/features/settings/components/SettingsDrawer.js

```diff
@@ -63,6 +74,24 @@ class SettingsDrawer extends Component<Props, *> {
         this._onNameFormSubmit = this._onNameFormSubmit.bind(this);
     }
 
+    /**
+     * Start Onboarding once component is mounted.
+     *
+     * NOTE: It autonatically checks if the onboarding is shown or not.
+     *
+     * @returns {void}
+     */
+    componentDidUpdate() {
+        if (this.props.isOpen) {
+
+            // TODO - Find a better way for this.
+            // Delay for 1 second to let drawer open.
```

Comment:
Is one full second necessary? Or just a 100ms setTimeout would do?

Reply:
Drawer animation is 220ms. So I guess we at least need that much.
On fast devices, 250ms works just fine. But on a little slow device 300ms sometimes does it correctly and sometimes the first onboarding created the problem which I mentioned earlier.

WDYT, how much should we set?

Reply:
Let's do 500, and please put that 220 in the comment so we remember :-)